content/docs/command-reference/stage/add.md

```diff
@@ -0,0 +1,432 @@
+# stage add
+
+Helper command to add or update <abbr>stages</abbr> in `dvc.yaml`. Requires a
+name and a command.
+
+## Synopsis
+
+```usage
+usage: dvc stage add [-h] [-q | -v] -n <name> [-d <path>] [-o <path>]
+                     [-O <path>] [-p [<path>:]<params_list>] [-m <path>]
+                     [-M <path>] [--plots <path>] [--plots-no-cache <path>]
+                     [-w <path>] [-f]
+                     [--outs-persist <path>] [--outs-persist-no-cache <path>]
+                     [--always-changed] [--external] [--desc <text>]
+                    command
+
+positional arguments:
+  command               Command to execute.
+```
+
+## Description
+
+`dvc stage add` is a helper for creating or updating
+[pipeline](/doc/command-reference/dag) stages in a `dvc.yaml` file (located in
+the current working directory).
+
+_Stages_ represent individual data processes, including their input and
+resulting outputs. They can be combined to capture simple data workflows,
+organize data science projects, or build detailed machine learning pipelines.
```

Comment:
Not sure we want to repeat this here again since it's already in the index and in the tooltip.